_posts/2019-06-1-bazel-0.25.md

```diff
@@ -0,0 +1,127 @@
+
+- [`--incompatible_disable_objc_library_resources`](https://github.com/bazelbuild/bazel/issues/7594) 
+
+- [`--incompatible_disable_legacy_crosstool_fields`](https://github.com/bazelbuild/bazel/issues/6861)
+
+- [`--incompatible_disable_expand_if_all_available_in_flag_set`](https://github.com/bazelbuild/bazel/issues/7008) 
+
+- [`--incompatible_disable_legacy_crosstool_fields`](https://github.com/bazelbuild/bazel/issues/6861)
+
+- [`--incompatible_no_attr_license`](https://github.com/bazelbuild/bazel/issues/6420) 
+
+- [`--incompatible_no_transitive_loads`](https://github.com/bazelbuild/bazel/issues/5636)
+
+- [`--incompatible_bzl_disallow_load_after_statement`](https://github.com/bazelbuild/bazel/issues/5815) 
+
+- [`--incompatible_disable_legacy_crosstool_fields`](https://github.com/bazelbuild/bazel/issues/6861) 
+
+- [`--incompatible_disable_expand_if_all_available_in_flag_set`](https://github.com/bazelbuild/bazel/issues/7008) 
+
+- [`--incompatible_merge_genfiles_directory`](https://github.com/bazelbuild/bazel/issues/6761)
+
+- [`--incompatible_disallow_load_labels_to_cross_package_boundaries`](https://github.com/bazelbuild/bazel/issues/6408)
+
+- [`--incompatible_use_jdk11_as_host_javabase`](https://github.com/bazelbuild/bazel/issues/7219)
+
+- [`--incompatible_disable_crosstool_file`](https://github.com/bazelbuild/bazel/issues/7320)
+
+- [`--incompatible_disable_legacy_cc_provider`](https://github.com/bazelbuild/bazel/issues/7036) 
+
+- [`--incompatible_disable_genrule_cc_toolchain_dependency`](https://github.com/bazelbuild/bazel/issues/6867) 
+
+- [`--incompatible_remove_cpu_and_compiler_attributes_from_cc_toolchain`](https://github.com/bazelbuild/bazel/issues/7075) 
+
+- [`--incompatible_use_toolchain_providers_in_java_common`](https://github.com/bazelbuild/bazel/issues/7186)
+
+- [`--incompatible_remote_symlinks`](https://github.com/bazelbuild/bazel/issues/7917)
+
+ 
+
+The following incompatible change flags have been added in this release and migration for them should begin. Check out the linked bugs to see when it is expected that the flags be turned on by default. 
+
+- [`--incompatible_windows_native_test_wrapper`](https://github.com/bazelbuild/bazel/issues/6622)
+
+- [`--incompatible_require_ctx_in_configure_features`](https://github.com/bazelbuild/bazel/issues/7793)
+
+- [`--incompatible_do_not_split_linking_cmdline`](https://github.com/bazelbuild/bazel/issues/7687)
+
+- [`--incompatible_string_join_requires_strings`](https://github.com/bazelbuild/bazel/issues/7802)
+
+- [`--incompatible_use_python_toolchains`](https://github.com/bazelbuild/bazel/issues/7899) and see [#7375](https://github.com/bazelbuild/bazel/issues/7375)
+
+
+## Android
+
+ - Android NDK C++ toolchain is now configured in Starlark
+
+ - Fix an issue where the Android resource processor did not surface
+
+    errors from aapt2 compile and link actions.
+
+
+## Other Changes
+
+  - Bazel is now ~20MiB smaller, from unbundling the Android rules'
```

Comment:
may be elaborating more about the new binary size @meisterT

Reply:
I think this is fine for now, we will have more comments the next two releases.